webpack/assets/javascripts/react_app/actions/StatiscticChartActions.jsx

```diff
@@ -0,0 +1,7 @@
+import API from "../API";
```

Comment:
Does this help?

https://github.com/facebook/flux/raw/master/docs/img/flux-diagram-white-background.png

It's from the facebook flux github repository, https://github.com/facebook/flux